tables/ar.tbl

```diff
@@ -16,3 +16,5 @@
 
 include ar-ar-g1.utb
 include braille-patterns.cti
+include ar-ext.cti
```

Comment:
The table name should probably be "ar-ext.uti", right? (The "C" stands for "contracted", the "U" for "uncontracted")